sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.digitaltwins.core.models.DigitalTwinsGetByIdResponse;
+import reactor.core.publisher.Mono;
+
+
+/**
+ * This class provides a client for interacting asynchronously with an Azure Digital Twins instance.
+ *
+ * <p>
+ * This client is instantiated through {@link DigitalTwinsClientBuilder}.
+ * </p>
+ *
+ * <p>
+ * This client allows for management of digital twins, their components, and their relationships. It also allows for managing
+ * the digital twin models and event routes tied to your Azure Digital Twins instance.
+ * </p>
+ */
+@ServiceClient(builder = DigitalTwinsClientBuilder.class)
+public class DigitalTwinsAsyncClient {
```

Comment:
Generally clients should be final as we don't normally allow them to be extended instead opting for a composition approach.